book/mars_maps.ipynb

```diff
@@ -0,0 +1,188 @@
+    "fig.colorbar(frame=['a5000','x+lElevation','y+lm'])\n",
+    "\n",
+    "fig.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e3530ac",
+   "metadata": {},
+   "source": [
+    "A very interesting feature is Mount Olympus (Olympus Mons - see more details at https://mars.nasa.gov/resources/22587/olympus-mons), centered at approximately 19°N and 133°W, with a height of 22 km (14 miles) and approximately 700 km (435 miles) in diameter. Let's use the original dataset at 32 pixels/degree resolution and plot a (not so interesting) map with `xarray`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01fac891",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dset_olympus = dset_mars.sel(latitude=slice(25,13),longitude=slice(210,240)).alt.astype(float)\n",
+    "dset_olympus.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fe62c4a7",
+   "metadata": {},
+   "source": [
+    "We use the same sequence as other pygmt tutorial notebooks to make a map."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f18b522",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = pygmt.Figure()\n",
+    "\n",
+    "fig.grdview(grid=dset_olympus,\n",
+    "           region=[210,240,13,25,-5000,23000],\n",
+    "           surftype='s',\n",
+    "           projection='M12c',\n",
+    "           perspective=[150,45],\n",
+    "           zsize='4c',\n",
+    "           frame=['xa5f1','ya5f1','z5000+lmeters','wSEnZ'],shading='+a50+nt1')\n",
+    "\n",
+    "bounds = [[210,13],\n",
+    "         [210,25],\n",
+    "         [240,25],\n",
+    "         [240,13],\n",
+    "         [210,13]]\n",
+    "\n",
+    "fig.colorbar(perspective=True,frame=['a5000','x+lElevation','y+lm'])\n",
+    "with fig.inset(position='JTR+w3.5c+o0.2c',margin=0,box=None):\n",
+    "    fig.grdimage(grid=dset_mars_topo,region='g',frame=True,projection='W3.5c')\n",
+    "    fig.plot(bounds, pen='1p,red')\n",
+    "\n",
+    "fig.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55a14a22",
+   "metadata": {},
+   "source": [
+    "And we're going to add some perspective, as well as a more interesting color scale. For ease of understanding, let's separate the region of interest with the same cutout that we created the base of the Olympus Mons topography dataset.\n",
+    "\n",
+    "**A few notes**\n",
+    "\n",
+    "`zsize` is a bit critical here because the volcano is very big (28 km if we consider -5000 to +23000 m). Likewise, `perspective=[150,45]` was choosen attempting (it's a matter of taste) and depends of which flank of the volcano you want to show. But this choice has to be made according to `shading` since to give a good 3D impression, the lighting must be adjusted according to the elevation and azimuth of the perspective. Finally, the pen outline is made smooth and small to enhance the contours of the topography.\n",
```

Comment:
```suggestion
    "`zsize` is a bit critical here because the volcano is very big (28 km if we consider -5000 to +23000 m). Likewise, `perspective=[150,45]` was chosen attempting (it's a matter of taste) and depends of which flank of the volcano you want to show. But this choice has to be made according to `shading` since to give a good 3D impression, the lighting must be adjusted according to the elevation and azimuth of the perspective. Finally, the pen outline is made smooth and small to enhance the contours of the topography.\n",
```